core/src/playn/core/Surface.java

```diff
@@ -379,6 +379,19 @@ public Surface drawLine (float x0, float y0, float x1, float y1, float width) {
     return this;
   }
 
+  /**
+   * Draws the specified rectangle.
+   * This is implemented by drawing four lines corresponding to the rectangle edges.
+   */
+  public Surface drawRect (float x, float y, float width, float height, float lineWidth) {
```

Comment:
I would have called the last parameter `thickness` but I kept it as `lineWidth` for consistence with the equivalent parameter in the `drawLine` methods.